mlir/unittests/Analysis/Presburger/SimplexTest.cpp

```diff
@@ -7,6 +7,7 @@
 //===----------------------------------------------------------------------===//
 
 #include "mlir/Analysis/Presburger/Simplex.h"
+#include "../utils.h"
```

Comment:
Can this path be made absolute?

Reply:
I would have to move this into the include directory. I am not certain how these kinds of things are usually done, but if you feel having them in the includes makes more sense, I can certainly do that.